coral-spark/src/main/java/com/linkedin/coral/spark/transformers/HiveUDFTransformer.java

```diff
@@ -27,10 +27,10 @@
  * we use this transformer to fall back to the original Hive UDF defined in
  * {@link com.linkedin.coral.hive.hive2rel.functions.StaticHiveFunctionRegistry}.
  * This is reasonable since Spark understands and has ability to run Hive UDF.
- * Check `CoralSparkTest#testFallBackToLinkedInHiveUDFTransformer()` for an example.
+ * Check `CoralSparkTest#testHiveUDFTransformer()` for an example.
```

Comment:
I am not sure if the Java doc makes a lot of sense. Could you revise the Java doc as well?

Reply:
Modified